doc-content/drools-docs/src/main/asciidoc/AuthoringAssets/rules-attributes-ref.adoc

```diff
@@ -107,7 +107,17 @@ Example: `lock-on-active true`
 Example: `ruleflow-group "GroupName"`
 
 |`dialect`
-|A string identifying either `JAVA` or `MVEL` as the language to be used for code expressions in the rule. By default, the rule uses the dialect specified at the package level. Any dialect specified here overrides the package dialect setting for the rule.
+a|A string identifying either `JAVA` or `MVEL` as the language to be used for code expressions in the rule. By default, the rule uses the dialect specified at the package level. Any dialect specified here overrides the package dialect setting for the rule.
 
 Example: `dialect "JAVA"`
+
+NOTE: When you use {PRODUCT} without an executable rule model, the consequences that are written using `dialect "JAVA"` are supported until JAVA 5 version. For more information about executable rule models, see
```

Comment:
'd write 

```"NOTE: When you use {PRODUCT} without the executable rule model, the consequences written using `dialect "JAVA"` support only Java 5 syntax"```

Executable model (not executable rule model) is not countable, so you can't use the "a" article. 
Java is upper case first
I'm not sure about the passive form "are supported"

Reply:
@lucamolteni 
thanks for your feedback. will make the changes accordingly. 

would you suggest removing the reference of executable rule model section too?

Reply:
Hi @hmanwani-rh - I need a bit more understanding regarding "consequences". Is this referring to "rule consequences"?

Reply:
@michelehaglund, yes, it is.

Reply:
Thank you for confirming @hmanwani-rh Let me know if this minor rewrite works. I was trying to avoid "that are written using dialect "JAVA" support only Java 5 syntax" 

When you use {PRODUCT} without the executable model, the dialect "JAVA" rule consequences only support the Java 5 language syntax. For more information about executable models, see Packaging and deploying a Red Hat Process Automation Manager project.

I am going to pass this along to Emily as she has been working to help me improve my editing skills. I am curious to hear her opinion on this one. Sorry, I don't mean to drag this out. She may feel that it was fine as you had it.

Reply:
Not a problem @michelehaglund, its good to have Emily's opinion. 
and thank you for your feedback, we can go for the rewrite, how about if I just remove the word language, since I believe it is self-explanatory. 
WDYT? 
tagging @emmurphy1 too.

Reply:
Sounds good to me @hmanwani-rh I can resolve the jira so that you can move forward. We can get @emmurphy1 s input when she has some time. A learning opportunity :)

Reply:
Hi @hmanwani-rh I agree. Sorrry for the churn.

Reply:
@michelehaglund, no problem